gradle.properties

```diff
@@ -1 +1,2 @@
-camelVersion = 2.24.2
\ No newline at end of file
+camelVersion  = 2.25.1
+intellij_type = IU
```

Comment:
Please remove this. It is default to IC, but we have make it possible to override locally